kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/build/BuildMojoTest.java

```diff
@@ -58,4 +73,35 @@ void buildServiceConfigBuilder_shouldReturnNonNullResourceConfigIfConfigured() {
             .returns("200m", c -> c.getLimits().get("cpu"))
             .returns("1Gi", c -> c.getRequests().get("memory"));
     }
+
+    @Test
+    @DisplayName("when Dockerfile present in project base directory, then generate opinionated ImageConfiguration for Dockerfile")
+    void simpleDockerfileModeWorksAsExpected(@TempDir File temporaryFolder) throws IOException {
```

Comment:
These tests (task + Mojo) work, but they aren't exactly where they should actually be.
I'll try to give take it for a spin and see if we can make this easier.